frontend/src/Components/AddedItems/AddedItems.jsx

```diff
@@ -47,9 +48,27 @@ function AddedItems() {
         }
     }
 
-    // Hàm xóa sản phẩm khỏi giỏ hàng
-    const handleRemove = (id) => {
-        dispatch(removeBook(id));
+    const handleRemove = async (id) => {
+      
+        const result = await Swal.fire({
```

Comment:
if we want to use toast, it must be defined again, it is not clear, let create reuse component to reuse toast